src/resultset/formatters/resultset_replyverbose.c

```diff
@@ -217,33 +218,43 @@ static void _ResultSet_VerboseReplyWithVector
 
 void ResultSet_EmitVerboseRow
 (
-	RedisModuleCtx *ctx,
-	bolt_client_t *bolt_client,
-	GraphContext *gc,
-	SIValue **row,
-	uint numcols
+	ResultSet *set,
+	SIValue **row
 ) {
+	RedisModuleCtx *ctx = set->ctx;
 	// Prepare return array sized to the number of RETURN entities
-	RedisModule_ReplyWithArray(ctx, numcols);
+	RedisModule_ReplyWithArray(ctx, set->column_count);
```

Comment:
if possible access `ResultSet` fields via functions, letting a reply formatter tinker with the `ResultSet` internals isn't recomendded